djangocms_picture/locale/en/LC_MESSAGES/django.po

```diff
@@ -17,80 +17,188 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: cms_plugins.py:12
-msgid "Picture"
+#: cms_plugins.py:19 models.py:68
+msgid "Image"
+msgstr ""
+
+#: cms_plugins.py:30
+msgid "Advanced settings"
+msgstr ""
+
+#: cms_plugins.py:40
+msgid "Link settings"
+msgstr ""
+
+#: cms_plugins.py:48
+msgid "Cropping settings"
 msgstr ""
 
 #: models.py:27
-msgid "left"
+msgid "Align left"
 msgstr ""
 
 #: models.py:28
-msgid "right"
+msgid "Align right"
 msgstr ""
 
 #: models.py:29
-msgid "center"
+msgid "Align center"
 msgstr ""
 
-#: models.py:32
-msgid "image"
+#: models.py:37
+msgid "Open in new window"
 msgstr ""
 
-#: models.py:34
-msgid "link"
+#: models.py:38
+msgid "Open in same window"
 msgstr ""
 
-#: models.py:35
-msgid "If present, clicking on image will take user to link."
+#: models.py:39
+msgid "Delegate to parent"
 msgstr ""
 
-#: models.py:38
-msgid "page"
+#: models.py:40
+msgid "Delegate to top"
 msgstr ""
 
-#: models.py:40
-msgid "If present, clicking on image will take user to specified page."
+#: models.py:46
+msgid "Default"
+msgstr ""
+
+#: models.py:62
+msgid "Template"
 msgstr ""
 
-#: models.py:44
-msgid "alternate text"
+#: models.py:75
+msgid "External image"
 msgstr ""
 
-#: models.py:45
+#: models.py:78
 msgid ""
-"Specifies an alternate text for an image, if the imagecannot be displayed."
-"<br />Is also used by search enginesto classify the image."
+"If provided, overrides the embedded image. Certain options such as cropping "
+"are not applicable for external images."
```

Comment:
*applicable to*

Also better to say exactly which options, not just "certain options".